repos/system_upgrade/el7toel8/actors/satelliteupgradefacts/actor.py

```diff
@@ -0,0 +1,26 @@
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRPM, RepositoriesSetupTasks, RpmTransactionTasks, SatelliteFacts
+from leapp.tags import IPUWorkflowTag, FactsPhaseTag
+
+
+class SatelliteUpgradeFacts(Actor):
+    """
+    No documentation has been provided for the satellite_upgrade_facts actor.
+    """
+
+    name = 'satellite_upgrade_facts'
+    consumes = (InstalledRPM, )
+    produces = (RepositoriesSetupTasks, RpmTransactionTasks, SatelliteFacts)
+    tags = (IPUWorkflowTag, FactsPhaseTag)
+
+    def process(self):
+        local_postgresql = has_package(InstalledRPM, 'rh-postgresql12-postgresql-server')
+        self.produce(SatelliteFacts(local_postgresql=local_postgresql))
```

Comment:
btw, why to produce new satellite facts if the only fact is get from other facts? this can be performed in the other actor as well. What is the added value with the new model in this case? basically the first line provides the same information as the second one.

Reply:
So far "because I wanted to play with facts and a boolean looked simple to act upon", but yeah, sure, you're right, there is right now nothing that consuming the `InstalledRPM` fact directly wouldn't do for us.